include/dogecoin/compat/byteswap.h

```diff
@@ -36,28 +36,24 @@
 // Non-Mac OS X / non-Darwin
 
 #if HAVE_DECL_BSWAP_16 == 0
-LIBDOGECOIN_API inline uint16_t bswap_16(uint16_t x) {
+LIBDOGECOIN_API inline uint16_t bswap_16(uint16_t x)
+{
     return (x >> 8) | ((x & 0x00ff) << 8);
 }
 #endif // HAVE_DECL_BSWAP16
 
 #if HAVE_DECL_BSWAP_32 == 0
-LIBDOGECOIN_API inline uint32_t bswap_32(uint32_t x) {
-    return (((x & 0xff000000U) >> 24) | ((x & 0x00ff0000U) >>  8) |
-            ((x & 0x0000ff00U) <<  8) | ((x & 0x000000ffU) << 24));
+LIBDOGECOIN_API inline uint32_t bswap_32(uint32_t x)
+{
+    return (((x & 0xff000000U) >> 24) | ((x & 0x00ff0000U) >> 8) |
+            ((x & 0x0000ff00U) << 8) | ((x & 0x000000ffU) << 24));
 }
 #endif // HAVE_DECL_BSWAP32
 
 #if HAVE_DECL_BSWAP_64 == 0
-LIBDOGECOIN_API inline uint64_t bswap_64(uint64_t x) {
-     return (((x & 0xff00000000000000ull) >> 56)
-          | ((x & 0x00ff000000000000ull) >> 40)
-          | ((x & 0x0000ff0000000000ull) >> 24)
-          | ((x & 0x000000ff00000000ull) >> 8)
-          | ((x & 0x00000000ff000000ull) << 8)
-          | ((x & 0x0000000000ff0000ull) << 24)
-          | ((x & 0x000000000000ff00ull) << 40)
-          | ((x & 0x00000000000000ffull) << 56));
+LIBDOGECOIN_API inline uint64_t bswap_64(uint64_t x)
+{
+    return (((x & 0xff00000000000000ull) >> 56) | ((x & 0x00ff000000000000ull) >> 40) | ((x & 0x0000ff0000000000ull) >> 24) | ((x & 0x000000ff00000000ull) >> 8) | ((x & 0x00000000ff000000ull) << 8) | ((x & 0x0000000000ff0000ull) << 24) | ((x & 0x000000000000ff00ull) << 40) | ((x & 0x00000000000000ffull) << 56));
```

Comment:
May still be OK to format these bit swaps on separate lines.

Reply:
reverted